src/BenchmarkDotNet/Validators/ISdkProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace BenchmarkDotNet.Validators
+{
+    public interface ISdkProvider
```

Comment:
`ISdkProvider` name implies that a different sdk besides dotnet could be used. Does that make sense? Does `RoslynToolchain` use an sdk that isn't dotnet?

Reply:
Also, I think this interface should be moved to `BenchmarkDotNet.Toolchains` namespace.